<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PilotDetails</class>
 <widget class="QDialog" name="PilotDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/qutescoop.png</normaloff>:/icons/qutescoop.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Pilot</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblPilotInfo">
          <property name="toolTip">
           <string>Link opens the VATSIM Statistics Center</string>
          </property>
          <property name="text">
           <string>&lt;a href=&quot;https://&quot;&gt;Surname Name, (EDDF)&lt;/a&gt;</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblConnected">
          <property name="text">
           <string>on SERVER for 1:41</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbAlias">
        <property name="toolTip">
         <string>Add or edit an alias for this VATSIM member</string>
        </property>
        <property name="text">
         <string>aliasâ€¦</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Aircraft</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Opens searchable list &amp;quot;ICAO Aircraft Type Designators&amp;quot; in web browser.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&lt;a href=&quot;https://www.icao.int/publications/DOC8643/Pages/Search.aspx&quot;&gt;type&lt;/a&gt;</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAircraft">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Added dynamically.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>A319</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open list of airline codes in web browser.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&lt;a href=&quot;https://en.wikipedia.org/wiki/List_of_airline_codes&quot;&gt;airline&lt;/a&gt;</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAirline">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Added dynamically&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Austrian</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,0,0,0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="margin" stdset="0">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>GS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblGroundspeed">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>419 kts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>alt</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAltitude">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>28000 ft</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>squawk:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSquawk">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;dynamically set&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>2200</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupStatus">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="lblFlightStatus">
        <property name="toolTip">
         <string>%: percent of route completion&lt;br&gt;
TTG: estimated remaining enroute time (&lt;b&gt;t&lt;/b&gt;ime &lt;b&gt;t&lt;/b&gt;o &lt;b&gt;g&lt;/b&gt;o)&lt;br&gt;
ETA: calculated &lt;b&gt;e&lt;/b&gt;stimated &lt;b&gt;t&lt;/b&gt;ime of &lt;b&gt;a&lt;/b&gt;rrival&lt;br&gt;
delay: difference between planned and calculated ETA</string>
        </property>
        <property name="text">
         <string>35%, TTG 0:33 hrs, ETA 1120z, Delay -0:05 hrs</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupFp">
     <property name="title">
      <string>Flightplan (IFR)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>departure aerodrome / &lt;b&gt;p&lt;/b&gt;lanned &lt;b&gt;t&lt;/b&gt;ime of &lt;b&gt;d&lt;/b&gt;eparture</string>
        </property>
        <property name="text">
         <string>dep/PTD</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>open departure aerodrome details dialog</string>
        </property>
        <property name="text">
         <string>LOWW (Vienna, AT)</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblPlanEtd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>ETD: &lt;b&gt;e&lt;/b&gt;stimated &lt;b&gt;t&lt;/b&gt;ime of &lt;b&gt;d&lt;/b&gt;eparture</string>
        </property>
        <property name="text">
         <string>1035</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>destination / &lt;b&gt;p&lt;/b&gt;lanned &lt;b&gt;t&lt;/b&gt;ime of &lt;b&gt;a&lt;/b&gt;rrival</string>
        </property>
        <property name="text">
         <string>dest/PTA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonDest">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>open destination aerodrome details dialog</string>
        </property>
        <property name="text">
         <string>LOWI (Innsbruck, AT)</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblPlanEta">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>ETA: &lt;b&gt;e&lt;/b&gt;stimated &lt;b&gt;t&lt;/b&gt;ime of &lt;b&gt;a&lt;/b&gt;rrival</string>
        </property>
        <property name="text">
         <string>1125</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;alternate aerodrome / fuel endurance&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>altern/fuel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonAlt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>open alternate aerodrome details dialog</string>
        </property>
        <property name="text">
         <string>LOWS (Salzburg, AT)</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="lblFuel">
        <property name="toolTip">
         <string>fuel on board</string>
        </property>
        <property name="text">
         <string>2:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>hrs</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>filed &lt;b&gt;t&lt;/b&gt;rue &lt;b&gt;a&lt;/b&gt;ir&lt;b&gt;s&lt;/b&gt;peed / cruise level / &lt;b&gt;p&lt;/b&gt;lanned &lt;b&gt;t&lt;/b&gt;ime &lt;b&gt;e&lt;/b&gt;nroute</string>
        </property>
        <property name="text">
         <string>TAS/FL/PTE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblPlanTas">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>ICAO flightplan format for true airspeed (TAS)</string>
          </property>
          <property name="text">
           <string>N420</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPlanFl">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>ICAO flightplan format for flight level</string>
          </property>
          <property name="text">
           <string>F340</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="lblPlanEte">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>ETE: &lt;b&gt;e&lt;/b&gt;stimated &lt;b&gt;t&lt;/b&gt;ime &lt;b&gt;e&lt;/b&gt;nroute</string>
        </property>
        <property name="text">
         <string>0:50</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>hrs</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>route</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLabel" name="lblRoute">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>filed route</string>
        </property>
        <property name="text">
         <string>&lt;code&gt;KAPPI UW705 BEROK UW653 TULSI KAPPI UW705 BEROK UW653  KAPPI UW705 BEROK UW653  KAPPI UW705 BEROK UW653 TULSI&lt;/code&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="cbPlotRoute">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plot route on map (requires navigation database). Also accessible via right click on the airplane's label on the map.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&amp;plot
route</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QLabel" name="lblPlotStatus">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="toolTip">
         <string>Extracted from the route with help of the navigation database</string>
        </property>
        <property name="text">
         <string>waypoints (calculated): ...</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>remarks</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QLabel" name="lblRemarks">
        <property name="text">
         <string>&lt;code&gt;PBN/A1B1D1O1S2 NAV/RNVD1E2A1 DOF/220222 REG/DABVM EET/EDUU0013 LOVV0029 LJLA0039 LDZO0044 LQSB0054 LYBA0109 BKPR0117 LWSS0124 LGGG0133 LTBB0212 LGGG0217 LCCC0235 HECC0255 OJAC0325 OEJD0334 OBBB0447 OMAE0508 RMK/TCAS /V/&lt;/code&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="verticalBottom">
     <item>
      <widget class="QPushButton" name="buttonShowOnMap">
       <property name="toolTip">
        <string>move to and and zoom in on this plane</string>
       </property>
       <property name="text">
        <string>show on &amp;map</string>
       </property>
       <property name="shortcut">
        <string>S</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddFriend">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add to / remove from friends list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>add &amp;friend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonFrom</tabstop>
  <tabstop>buttonDest</tabstop>
  <tabstop>buttonAlt</tabstop>
  <tabstop>buttonShowOnMap</tabstop>
  <tabstop>buttonAddFriend</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonShowOnMap</sender>
   <signal>clicked()</signal>
   <receiver>PilotDetails</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>PilotDetails</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
